### ISSUE TITLE
NullReferenceException in Server.Completion

### ISSUE THREAD

**Issue by jakebailey** [Microsoft/python-language-server#865]
Seeing this one at the top now.

```
   at Microsoft.Python.LanguageServer.Implementation.Server.Completion(CompletionParams params, CancellationToken cancellationToken)
   at Microsoft.Python.LanguageServer.Implementation.LanguageServer.Completion(JToken token, CancellationToken cancellationToken)
```

**Comment by MikhailArkhipov** [comment]
This should be fixed in #837 (IC exception)

**Comment by jakebailey** [comment]
The latest version has that merged, so this is some other nullref. Not really sure what it could be, given it's supposedly happening in that function directly.
